code/datums/diseases/_MobProcs.dm

```diff
@@ -17,7 +17,7 @@
 	if(HasDisease(D))
 		return 0
 
-	if(count_by_type(viruses, /datum/disease/advance) >= 3)
```

Comment:
You need to include this in your changelog. If you're going to make engineered diseases no longer prevent other diseases, you might as well remove the entire `if` statement line here.

This effects gameplay, why _didn't_ you note this in the body of the PR?

Reply:
This is a change that explicitly exists to limit the number of advanced diseases to 1 as opposed to 3---which was documented.

Reply:
Guess I'm not all that great at reading code no? 😅

Reply:
Thinking on it though, this does make me wonder something. 

I believe the intent of this code is to prevent stacking of multiple advance level disease. In practice though, I think it *may* be also blocking acquiring of *any* disease at all if you have 3 advance diseases...so at the end of the day, thanks for mentioning it; this is something I'll need to test (and probably adjust if it works a particular way).

Reply:
Yeah, glad you mentioned this, because it got me to look into a few things and test things more closely.

So having advanced viruses up to the limit blocks *all* further virus infections; not just more advanced viruses; if you had 3 advanced viruses and someone penned you with prions or what have you? You'd be immune to the disease.

Don't think that'd the intention of the code, but it's certainly what it does....adjusting.